packages/lesswrong/components/dialogues/ActiveDialogues.tsx

```diff
@@ -57,8 +86,18 @@ export const ActiveDialogues = ({classes}: {
 
   const { captureEvent } = useTracking(); //it is virtuous to add analytics tracking to new components
   const [activeDialogues, setActiveDialogues] = useState<ActiveDialogue[]>([]);
-
   const currentUser = useCurrentUser();
+  const updateCurrentUser = useUpdateCurrentUser();
+  const [isExpanded, setIsExpanded] = useState<boolean>(!currentUser?.hideActiveDialogueUsers);
+  const toggleExpanded = async () => {
```

Comment:
This doesn't need to be async if you aren't awaiting `updateCurrentUser` (rather than `void`)